src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,120 @@
+import React from "react"
+import PropTypes from "prop-types"
+import styles from './Carrousel.module.scss'
+
+class CarouselElement extends React.Component {
```

Comment:
Any particular reason for using class components? its a good way to understand react lifecycle phases, but Hooks are the preferred new way for new projects, any wrong with using class, but its good to understand hooks as many projects uses it

Reply:
I did not watch the hooks workshop. most of the classes where migrated to hooks weeks ago. some of them are still pending